web/packages/teleport/src/Discover/Database/SetupAccess/SetupAccess.tsx

```diff
@@ -95,7 +94,17 @@ export function SetupAccess(props: State) {
   }
 
   function handleOnProceed() {
-    onProceed({ databaseNames: selectedNames, databaseUsers: selectedUsers });
+    let numStepsToSkip;
+    // Skip test connection since test connection currently
+    // only supports one resource testing and auto enrolling
+    // enrolls resources > 1.
+    if (agentMeta.autoDiscoveryConfig) {
+      numStepsToSkip = 2;
```

Comment:
so i messed up how this should work. originally nextStep() meant go to next step (which is currentStep +1). 
https://github.com/gravitational/teleport/blob/da589355d4ea55de276062db09f440c6fefdb2d6/web/packages/teleport/src/Discover/useDiscover.tsx#L321

then came the need to add ability to skip, and instead of passing in the number of steps to `skip`, i passed in the number of total increments. so `nextStep(2)` means skip only one step.

and this vairable name is misleaing, i renamed to `numStepsToIncrement`